docs/en/integrations/index.mdx

```diff
@@ -147,7 +147,7 @@ We are actively compiling this list of ClickHouse integrations below, so it's no
 |EMQX|<EmqxSVG style={{width: '3rem'}} />|Data ingestion|EMQX is an open source MQTT broker with a high-performance real-time message processing engine, powering event streaming for IoT devices at massive scale.|[Documentation](/docs/en/integrations/emqx)|
 |Explo|<img src={require('./images/logos/explo.png').default} class="image" alt="Explo logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data visualization|Explo is a customer-facing analytics tool for any platform|[Documentation](/docs/en/integrations/explo)|
 |Gigasheet|<img src={require('./images/logos/gigasheet.png').default} class="image" alt="Gigasheet logo" style={{width: '3rem', 'background-color': 'transparent'}}/>|Data visualization|A cloud big data analytics spreadsheet that enables business users to instantly analyze and explore ClickHouse data.|[Website](https://gigasheet.com/enterprise)|
-|Grafana|<GrafanaSVG style={{width: '3rem'}} />|Data visualization|With Grafana you can create, explore and share all of your data through dashboards.|[Documentation](/docs/en/integrations/data-visualization/grafana-and-clickhouse.md)|
+|Grafana|<GrafanaSVG style={{width: '3rem'}} />|Data visualization|With Grafana you can create, explore and share all of your data through dashboards.|[Documentation](/docs/en/integrations/data-visualization/grafana/index.md)|
```

Comment:
@Ryado maybe it's time to move it to the `Core integration`?

Reply:
Sounds good 👍

Reply:
There's a section at the top of this page for definitions:

---

ClickHouse integrations are organized by their support level:
- **Core integrations:** built or maintained by ClickHouse, they are supported by ClickHouse and live in the ClickHouse GitHub organization
- **Partner integrations:** built or maintained, and supported by, third-party software vendors
- **Community integrations:** built or maintained and supported by community members. No direct support is available besides the public GitHub repositories and community Slack channels

---

There's some overlap on the criteria, I could see it belonging to either. The only thing it doesn't match for core is that it doesn't belong to our ClickHouse GitHub org (Grafana owns the main repo)

Reply:
still, it's supported by ClickHouse team mostly